codespell_lib/data/dictionary.txt

```diff
@@ -42712,6 +42722,7 @@ reconncted->reconnected
 reconncting->reconnecting
 reconncts->reconnects
 reconsidder->reconsider
+reconsilded->reconciled
```

Comment:
Could you add other verbal forms (`reconsilde`, `reconsildes` and `reconsilding`)?